lib/open_project/costs/patches/query_patch.rb

```diff
@@ -79,7 +79,8 @@ def available_work_package_filters_with_costs
           'cost_object_id' => {
             type: :list_optional,
             order: 14,
-            values: CostObject.all(conditions: ['project_id IN (?)', project], order: 'subject ASC').map { |d| [d.subject, d.id.to_s] }
+            values: CostObject.where(['project_id IN (?)', project])
+                    .order('subject ASC').map { |d| [d.subject, d.id.to_s] }
```

Comment:
I had to look it up last week too, because I didn't know if multi column pluck was introduced with Rails 4.0 or 4.1 :)